conda_build/exceptions.py

```diff
@@ -0,0 +1,46 @@
+import textwrap
+SEPARATOR = "-" * 70
+
+indent = lambda s: textwrap.fill(textwrap.dedent(s))
+
+
+class CondaBuildException(Exception):
```

Comment:
@asmeurer these aren't caught by the user, it's handled inside `conda_build.main_build.main` (see the diff below).  FWIW, I thought this has been merged and released already.  I'm going to go ahead and merge it since changing from `Exception` to `RuntimeException` isn't a backwards incompatible change.  I'm -0 on the change, but if you want to do it, go for it.

I'd love to get this released.  Having a friendly error message instead of a stack trace would be a pretty big improvement.